.circleci/config.yml

```diff
@@ -290,7 +290,8 @@ jobs:
           command: |
             export PROJECT=`gcloud config get-value project`
             export ING_IP=$(kubectl get ing -o=jsonpath='{.items[0].status.loadBalancer.ingress..ip}' -n mgt-api)
-            helm ls --short | xargs -L1 helm --debug delete
+            cd ~/inference-model-manager/installer
+            ./uninstaller.sh
             sleep 3m
```

Comment:
maybe this `sleep 3` could be added into this script? if it is connected to the uninstalling process

Reply:
This sleep is added because clusters on Google Cloud create some additional resource. So in another clouds or bare metal environment this can be redundant